### PR TITLE
Added variable short formal names, removed french, added max/min length

### PR DESCRIPTION
**Pull request by mathew-thomson**
Added short formal names that include the short formal table names using the format "tableShortFormalName_variableShortFormalName" - perhaps considered a uniqueID?

variableShortFormalName varies from two to four characters. I don't think it's fully necessary for there to be a set character number, but if I'm mistaken I can revise. The general structure is to use the first two letters for sing word variable names, or up to four first letters from multi-word variable names. Rules were sometimes slightly bent to avoid duplication.

Removed the french language information in preparation for multi-language functionality.

Added columns for max and min value length, taken from validation/validate.csv in the validation branch. This validation file can likely be deleted now. I will go through and populate max and min values for length now by hand.

### CODE REVIEW

File: site/Tables.csv [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-MeasureClassHasUnit,Measure class has unit,Classe de mesure utilise unité,Shows relations between a measure class and reporting units.,Montre les relations entre les classes de mesures et les unités utilisées pour raporter leur valeur.
-MeasureGroup,Measure group,Groupe de mesure,Groups measures that have relate to the same.,
-UnitScale,Unit Scale,Échelle des unités,"Cardinality of a measurement (qualitative, quantitative, etc.).","Cardinalité d'une mesure (quantitative, qualitative, etc.)."
-Specimen,Specimen,Spécimen,Class of entity from which a measure is taken.,"Classe d'entité sur laquelle une mesure est prise (échantillon, site, etc.)."
-Unit,Unit of measure,Unité de mesure,Unit of reporting for values,Unité pour rapporter des valeurs
-UnitHasAggregationScale,Unit has aggregation scale,Unité utilise échelle d'aggrégation,Aggregations that are meaningful for a given unit,Aggrégations qui sont utilisables avec une unité
-AggregationScale,Aggregation scales,Échelle d'aggrégation,Scale used to aggregate values together,
\ No newline at end of file
+tableName,tableShortFormalName,tableLabel,tableDesc
```
Comment (DougManuel) [inline site/Tables.csv:1]: Looks good. I wonder if we should have `Sa` as opposed to `sa` to keep the Pascal case approach for the original table.

File: site/Variables.csv [hunk 2]
```diff
@@ -1,153 +1,153 @@
...
-Unit,latexEpression,,,,,,string,,
-Unit,descriptionEN,,,,,,string,,
-Unit,descriptionFR,,,,,,string,,
-Unit,unitScaleID,,,,UnitScale,unitScaleID,string,,
-Unit,aggregationScaleID,,,,AggregationScale,aggregationScaleID,string,,
-Unit,dataType,,,,,,string,,
-UnitHasAggregation,unitID,,,,Unit,unitID,string,,
-UnitHasAggregation,aggregationID,,,,Aggregation,aggregationID,string,,
\ No newline at end of file
+tableName,variableName,variableShortFormalName,variableLabel,key,foreignKeyTable,foreignKeyVariable,variableType,minlength,maxlength,variableDesc
```
Comment (DougManuel) [inline site/Variables.csv:1]: Suggest 'uid' or 'id' for `variableShortFormalName`.

**Review by DougManuel — CHANGES_REQUESTED** [review]
Just two a few small comments.

**Comment by DougManuel** [comment]
This PR addresses issue https://github.com/Big-Life-Lab/ODM/issues/181.

File: site/Tables.csv (hunk 1 above)
Comment (mathew-thomson) [reply site/Tables.csv:1]: I would say I'm pretty agnostic on the question of capitalization - my thought process for all lowercase is that capitalization in Pascal Case or Camel Case seems to be to improve readability in the absence of spaces, but I don't know that the UIDs are all that readable regardless. Do you think I should update them all with a capitalized first letter?

File: site/Variables.csv (hunk 2 above)
Comment (mathew-thomson) [reply site/Variables.csv]: Done!

**Comment by mathew-thomson** [comment]
Updated the short formal name columns to uid and finished populating the max and min value fields - will work on putting up a breakdown of the naming methodology asap.

**Review by DougManuel — APPROVED** [review]
I'm going to approve this PR and merge it. We can have further discussions and review within the larger VOC-variables PR.

@mathew-thomson we basically need an uid for every element in the ODM (variables, measures, tables, and variable categories). I suggest that you:
1. add uid to variables.csv and most other tables.
1. hold of adding uid to variable_details.csv until we have a discussion.

In adding uids, we may need to have a discussion about how we generate a uid. 

I am concerned that it may be difficult to where a unique ID come from for some tables, such as the measure table, or variableCategories, if we keep those.

**Pull request merged**